projects/igniteui-angular/src/lib/grids/filtering/advanced-filtering/advanced-filtering-dialog.component.html

```diff
@@ -183,29 +183,21 @@ <h6 class="igx-filter-empty__title">
                            [(ngModel)]="searchValue"/>
                 </igx-input-group>
 
-                <igx-date-picker *ngIf="selectedColumn && selectedColumn.dataType === 'date'"
-                                 mode="dropdown"
-                                 [(ngModel)]="searchValue"
-                                 [locale]="grid.locale"
-                                 [outlet]="grid.outlet">
-                    <ng-template igxDatePickerTemplate let-openDialog="openDialog" let-value="value">
-                        <igx-input-group #dropDownTarget type="box" [displayDensity]="'compact'">
-                            <input #searchValueInput
-                                    igxInput
-                                    tabindex="0"
-                                    (keydown)="invokeClick($event)"
-                                    (click)="openDialog(dropDownTarget.element.nativeElement)"
-                                    [placeholder]="grid.resourceStrings.igx_grid_filter_row_date_placeholder"
-                                    autocomplete="off"
-                                    [value]="
-                                        selectedColumn.formatter
-                                        ? (value | columnFormatter:selectedColumn.formatter)
-                                        : (value | date:selectedColumn.pipeArgs.format:undefined:grid.locale)
-                                    "
-                                    [readonly]="true"
-                                    [disabled]="!selectedColumn || !selectedCondition || (selectedColumn && selectedColumn.filters.condition(selectedCondition).isUnary)"/>
-                        </igx-input-group>
-                    </ng-template>
+                <igx-date-picker #datePicker *ngIf="selectedColumn && selectedColumn.dataType === 'date'"
```

Comment:
The date picker looks different. It seems that the box input type is not applied. Also even when the date picker is disabled if you click it, it gets focused.
![image](https://user-images.githubusercontent.com/43128948/114701393-6e066700-9d2b-11eb-8e3c-a096ec64c8ff.png)